requirements/extra.txt

```diff
@@ -8,3 +8,4 @@ hydra-core>=1.0.5
 jsonargparse[signatures]>=4.6.0
 gcsfs>=2021.5.0
 rich>=10.2.2,!=10.15.*
+hivemind>=1.0.1
```

Comment:
@Borda Have we decided on adding a separate requirements file for strategies?

Reply:
#12723